torchgeo/trainers/classification.py

```diff
@@ -90,9 +101,12 @@ def __init__(self, **kwargs: Any) -> None:
             in_channels: Number of input channels to model
             learning_rate: Learning rate for optimizer
             learning_rate_schedule_patience: Patience for learning rate scheduler
+            freeze_backbone: Fine-tune the cls head by freezing the backbone
```

Comment:
```suggestion
            freeze_backbone: Fine-tune the classifier head by freezing the backbone
```

Reply:
linear probing is the action of actually fine-tuning. freezing the backbone is what is happening when you set this arg to True. Imo makes more sense to call it `freeze_backbone`.